instrumentation/src/main/java/io/opentelemetry/android/instrumentation/startup/InitializationEvents.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.startup;
+
+import io.opentelemetry.android.OtelAndroidConfig;
+
+public interface InitializationEvents {
+
+    void sdkInitializationStarted();
```

Comment:
Since we have this event, we should probably also include one for when the initialization has finished.

Reply:
Yeah totally....this thing is just a shell/stub right now.